### ISSUE TITLE
Duplicate tag in fuzzer 047

### ISSUE THREAD

**Issue by kgugala** [SymbiFlow/prjxray#992]
on current master (commit 693c621e9640589138683273c395c2a0f12d993e). Fuzzer 047 produces duplicate tags:

```
WARNING: got duplicate tag HCLK_IOI3.HCLK_IOI_LEAF_GCLK_BOT1.HCLK_IOI_CK_IGCLK6
  Orig line: HCLK_IOI3.HCLK_IOI_LEAF_GCLK_BOT1.HCLK_IOI_CK_IGCLK6 27_184 31_189
  New line : HCLK_IOI3.HCLK_IOI_LEAF_GCLK_BOT1.HCLK_IOI_CK_IGCLK6 27_184 31_184 31_189
```

**Comment by litghost** [comment]
I believe this is related to the fake that 000 was missing an hclk_ioi3 entry.  Potentially fixed in #985

**Comment by tmichalak** [comment]
Seems #985 fixed the issue. Closing.
